docs/samples/browser-plugin-meetings/index.html

```diff
@@ -78,12 +78,14 @@ <h2 class="collapsible">Meeting Management</h2>
                 <legend>Environment</legend>
                 <input type="checkbox" id="integration-env">
                 <label for="integration-env">Enable Integration Environment</label><br>
+                <input type="checkbox" id="enable-fedramp">
+                <label for="enable-fedramp">Enable Fedramp</label>
               </div>
 
               <div>
-                <legend>FedRAMP</legend>
```

Comment:
why removing the fedramp check ?/

Reply:
I didn't remove it, it's still there, I just moved it up to the environment section:
![image](https://github.com/webex/webex-js-sdk/assets/7473332/444e3bc8-2ade-4476-869a-5458decda575)

Reply:
actually, now after merging in latest beta, the UI looks a bit different, so I've kept Fedramp in the same position as it is and removed turn discovery checkbox as turn discovery config option has already been removed a few weeks ago, so now the sample app looks like this:
![image](https://github.com/webex/webex-js-sdk/assets/7473332/72d3e006-01dd-4621-b08d-8d24a7281eed)